src/App.js

```diff
@@ -1,11 +1,71 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
 import './App.css';
 
 const App = () => (
-  <div className="App">
-    <Calculator />
-  </div>
+  <Router>
+    <header className="header">
+      <h1>Math Magician</h1>
+      <ul className="navigation">
+        <li><a href="home">Home</a></li>
+        <li><a href="calculator">Calculator</a></li>
+        <li><a href="quote">Quote</a></li>
```

Comment:
- [ ] Your app **refreshes** when a menu link of a page is clicked. Kindly ensure that your app conforms to the SPA. You can improve upon this aspect by replacing the `<a>` tags with react `Link` tags.